src/zone.rs

```diff
@@ -65,13 +65,14 @@ pub struct Zone {
     pub boundary: Option<geo::MultiPolygon<f64>>,
 
     pub tags: Tags,
+    pub center_tags: Tags,
```

Comment:
you need to add `#[serde(default)]` not to break the retrocompatibility with `cosmogony2mimir`